src/image-sync-manager.ts

```diff
@@ -51,6 +54,9 @@ export class ImageSyncManager {
   private _onFirstPlaying = () => {
     this._firstPlaying = true;
     this._imagePlayer.preLoadImages();
+    if (this._imagePlayer.images.length > 0) {
+      this._onSlidesInitialized();
+    }
```

Comment:
Why not call this on line 99 where it already checks if there are slides?
`    if (slides.length && this._firstPlaying) {
      this._imagePlayer.preLoadImages();
    }`

Reply:
firstPlaying event is triggered **after** timedMetadataAdded. this condition never happens... which in general is a bug i guess.